docs/src/modules/utils/useCustomizationPlayground.tsx

```diff
@@ -244,12 +248,26 @@ const getCodeExample = ({
   }
 
   if (selectedCustomizationOption === 'sxProp') {
-    code = `${code}\n<${componentName}${formatComponentProps(examples.componentProps, 1)}
+    if (selectedExample?.parentSlot) {
+      const componentProps = {
+        ...examples.componentProps,
+        slotProps: {
+          ...examples.componentProps?.slotProps,
+          [selectedExample?.parentSlot]: {
+            sx: { [`'& .Mui${selectedDemo}-${selectedSlot}'`]: tokens },
```

Comment:
WDYT about removing the `& ` prefix?
I'm not sure if it serves any purpose in such cases. 🙈 
```suggestion
            sx: { [`'.Mui${selectedDemo}-${selectedSlot}'`]: tokens },
```